lib/mpl_toolkits/tests/test_axisartist_axis_artist.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import matplotlib.pyplot as plt
+from matplotlib.testing.decorators import image_comparison
+
+from mpl_toolkits.axisartist import AxisArtistHelperRectlinear
+from mpl_toolkits.axisartist.axis_artist import (AxisArtist, AxisLabel,
+                                                 LabelBase, Ticks, TickLabels)
+
+
+@image_comparison(baseline_images=['axis_artist_ticks'],
+                  extensions=['png'], style='default')
+def test_ticks():
```

Comment:
This test fails because ticks provided by axis-artist are clipped by the gc to the Axes; should I just disable that clipping? What about when ticks are set outside the view limits? Should it do clipping based on that instead?

Reply:
This is fixed by d8ada7a.